src/locales/en/translation.json

```diff
@@ -24,6 +24,8 @@
   "loadmap": "Load Map",
   "noappointments": "No appointments were found near {{postalCode}}. Please try again later.",
   "password": "Password",
+  "popupwarning": "This is a pop-up clinic",

```

Comment:
Should be translated into other languages.

Reply:
Are we going to wait until we have the translations before we merge, or can we merge and cut a new ticket to track that work?

Reply:
it's not a big deal, would you like me to update the branch 106-add-popup-warning to add the French? Just make sure to pull and merge conflicts afterwards.

Reply:
added the French translations

Reply:
we should start a new ticket for the rest of the translations

Reply:
Created a ticket https://github.com/Vaccine-Hunters-Canada/VaxFinder-Frontend/issues/110